molecule/topic_data_creation.yml

```diff
@@ -17,5 +18,5 @@
     - name: Create Topic Data
       shell: |
         seq 10 | kafka-console-producer --topic test-replicator-source \
-         --broker-list mds-kafka-broker1:9091 --producer.config /etc/kafka/client.properties
+         --broker-list mds-kafka-broker1:9092 --producer.config /etc/kafka/client.properties
```

Comment:
dont change this line, the client.properties file is written to match a specific listener

Reply:
Done